scripts/deployment/deploy.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import dataclasses
+import argparse
+import logging
+import os
+import psycopg2
+import copy
+
+
+@dataclasses.dataclass
+class PostgresDBConn:
+    """This dataclass contains all information related to making a connection to Postgres DB."""
+    username: str
+    password: str
+    host: str
+    database: str
+    port: int
+
+
+def parse_args() -> argparse.Namespace:
+    """CLI args parsing function."""
+    parser = argparse.ArgumentParser(
+        description="Deploys structures to DB. (script version: 1.0)",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument(
+        "-ph", "--host",
+        help="database server host (default: \"localhost\")",
+        default="localhost",
+    )
+    parser.add_argument(
+        "-p", "--port",
+        help="database server port (default: \"5432\")",
+        default="5432",
+    )
+    parser.add_argument(
+        "-d", "--dbname",
+        help="database name to connect to (default: \"ursa_unify_db\")",
+        required=True,
+    )
+    parser.add_argument(
+        "-U", "--username",
+        help="database user name, should be a high privileged account (default: \"postgres\")",
+        default="postgres",
+    )
+    parser.add_argument(
+        "-W", "--password",
+        help="database user password",
+        required=True,
+    )
+    parser.add_argument(
+        "-dir", "--dir",
+        help="the directory of database source files (default: current directory)",
+        default=os.getcwd()
+    )
+    parser.add_argument(
+        "--create-db",
+        action="store_true",
+        help="creates the target database (runs the scripts in the source root starting with '00_', which is/are expected creating the db",
+    )
+
+    return parser.parse_args()
+
+
+def execute_sql(conn_config: PostgresDBConn, sql: str):
+    conn = psycopg2.connect(
+        database=conn_config.database,
+        user=conn_config.username,
+        password=conn_config.password,
+        host=conn_config.host,
+        port=conn_config.port
+    )
+
+    conn.autocommit = True
+    cursor = conn.cursor()
+
+    cursor.execute(sql)
+
+    conn.commit()
+    conn.close()
+
+
+def read_file(file_name: str) -> str:
+    logging.debug(f"    - reading file `{file_name}`")
+    file = open(file_name, "r")
+    result = file.read()
+    file.close()
```

Comment:
Let's use context manager for opening and closing files - it will be handled properly even in case of a failure.

Somethig like 

```
with open(file_name, "r") as file:
    return file.read()
```